mne/channels/layout.py

```diff
@@ -931,3 +932,80 @@ def apply_montage(info, montage):
         raise ValueError('None of the sensors defined in the montage were '
                          'found in the info structure. Check the channel '
                          'names.')
+
+
+def generate_2d_layout(xy, w=.07, h=.05, pad=.02, ch_names=None,
+                       ch_indices=None, name='ecog', bg_image=None):
+    '''Generates a 2-D layout for plotting with plot_topo methods and
+    functions. XY points will be normalized between 0 and 1, where
+    normalization extremes will be either the min/max of xy, or
+    the width/height of bg_image.
+
+    Parameters
+    -------
+    xy : ndarray (N x 2) | 'interactive'
+        The xy coordinates of sensor locations. If 'interactive',
+        bg_image must not be None, as this will be displayed
+        and the user must click on positions of the image to
+        create xy values.
+
+    w, h : float
+        The width and height of each sensor's axis (between 0 and 1)
+
+    bg_image : str | ndarray
+        The image over which sensor axes will be plotted. Either a path to an
+        image file, or an array that can be plotted with plt.imshow. If
+        provided, xy points will be normalized by the width/height of this
+        image. If not, xy points will be normalized by their own min/max.
```

Comment:
Ahh - thank you for the clarification. I moved the import and updated the docstring.